lib/src/SearchBar.js

```diff
@@ -82,11 +93,15 @@ export default class SearchBar extends Component {
                         textInputRef = c;
                       }}
                       autoFocus={autoFocus}
+                      defaultValue={textInputDefaultValue}
                       value={textInputValue}
                       placeholder={placeholder}
                       onChangeText={onChangeText}
                       placeholderTextColor={fontColor}
                       style={[textInputStyle(fontSize, fontColor)]}
+                      returnKeyType="search"
+                      underlineColorAndroid="#FFF"
```

Comment:
We do not need to set them, I'm going to make it much more dynamic with passing all props. (aka. `...props`). Do not touch it, I will change it while merging :) Good job!